app/classifier/index.cjsx

```diff
@@ -5,6 +5,7 @@ ClassificationSummary = require './classification-summary'
 {Link} = require 'react-router'
 tasks = require './tasks'
 drawingTools = require './drawing-tools'
+{session} = require '../lib/session'
```

Comment:
I don't think that'd necessarily be better. The expiry time for the key would still be arbitrary. If you want to keep tracking sessions after a refresh you could look for classifications from the same ip address after a short time.